src/components/QueryEditor/types.ts

```diff
@@ -44,14 +48,19 @@ export interface SQLOptions extends SQLConnectionLimits, DataSourceJsonData {
   timeInterval: string;
 }
 
+// Match the Enums Expected in SqlUtil and SqlDS
+// https://github.com/grafana/grafana-plugin-sdk-go/blob/main/data/sqlutil/query.go#L18-L29
 export enum QueryFormat {
-  Timeseries = 'time_series',
-  Table = 'table',
+  Timeseries,
+  Table,
+  Logs,
+  Trace,
+  OptiuonMulti,
```

Comment:
This is the actual change, seems like `prettier` or `eslint` did their magic on save on the files for me